tests/jballerina-unit-test/src/test/resources/test-src/jvm/type-test-expr.bal

```diff
@@ -131,3 +131,29 @@ function testAnyJsonTypes() returns [boolean, boolean, boolean, boolean] {
 
     return [aa is json, bb is json, dd is json, ff is json];
 }
+
+function testIsLike1() {
```

Comment:
Can we rename this to reflect what's being tested?

e.g.,
```suggestion
function testIsLikeForTupleWithRestDescriptor() {
```

Reply:
Done